tools/booster/Proxy.hs

```diff
@@ -179,7 +179,12 @@ respondEither mbStatsVar booster kore req = case req of
                                 -- if we made one step, add the number of
                                 -- steps we have taken to the counter and
                                 -- attempt with booster again
-                                Log.logInfoNS "proxy" "kore depth-bound, continuing"
+                                when (koreResult.depth == 0) $ error "Expected kore-rpc to take at least one step"
```

Comment:
Are we _sure_ that in all `aborted` cases `kore-rpc` is able to take a step?
(Maybe `booster` is unable to detect that the execution is stuck, is that possible?)
What should definitely not happen is `reason = depth-bound` _and_ `depth == 0`